docs/keycodes.md

```diff
@@ -347,6 +347,14 @@ See also: [Leader Key](feature_leader_key.md)
 |---------|------------------------|
 |`KC_LEAD`|Begins a leader sequence|
 
+## Compose Onboard :id=compose-onboard
+
+See also: [Compose Onboard](feature_compose_onboard.md)
+
+|Key                 |Aliases    |Description                              |
+|--------------------|-----------|-----------------------------------------|
+|`KC_COMPOSE_ONBOARD`|`KC_COMPOB`|Begins Composing directly on the keyboard|
```

Comment:
```suggestion
|`KC_COMPOSE_ONBOARD`|`KC_CMOB`|Begins Composing directly on the keyboard|
```

We like these keycode aliases to be seven characters or less, when possible.

Also, this entire section should be moved up in the document to be between the Bluetooth and Dynamic Macros sections, as this page is organized alphabetically (with the exception of the Basic and Quantum sections).

Reply:
IMO Quantum keycodes should not have the `KC_` prefix either, as that is reserved for the basic keycode set.